pkg/doc/presexch/api.go

```diff
@@ -163,7 +63,7 @@ func WithJSONLDDocumentLoader(l ld.DocumentLoader) MatchOption {
 }
 
 // Match returns the credentials matched against the InputDescriptors ids.
-func (p *PresentationDefinition) Match(vp *verifiable.Presentation, // nolint:gocyclo,funlen
+func (pd *PresentationDefinition) Match(vp *verifiable.Presentation, // nolint:gocyclo,funlen
```

Comment:
what if multiple credentials matched??

Reply:
It is an old function. Will be rewritten soon.